File: Pidev_EGOV/EGOV_symfony/src/MyApp/UserBundle/Entity/ActesJuridiques.php
<?php

namespace MyApp\UserBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * ActesJuridiques
 *
 * @ORM\Table(name="actes_juridiques")
 * @ORM\Entity
 */
class ActesJuridiques
{
    /**
     * @var string
     *
     * @ORM\Column(name="titre_acte", type="string", length=100, nullable=true)
     */
    private $titreActe;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_acte", type="date", nullable=true)
     */
    private $dateActe;

    /**
     * @var string
     *
     * @ORM\Column(name="desc_acte", type="text", length=65535, nullable=true)
     */
    private $descActe;

    /**
     * @var integer
     *
     * @ORM\Column(name="id_acte", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idActe;



    /**
     * Set titreActe
     *
     * @param string $titreActe
     * @return ActesJuridiques
     */
    public function setTitreActe($titreActe)
    {
        $this->titreActe = $titreActe;

        return $this;
    }

    /**
     * Get titreActe
     *
     * @return string 
     */
    public function getTitreActe()
    {
        return $this->titreActe;
    }

    /**
     * Set dateActe
     *
     * @param \DateTime $dateActe
     * @return ActesJuridiques
     */
    public function setDateActe($dateActe)
    {
        $this->dateActe = $dateActe;

        return $this;
    }

    /**
     * Get dateActe
     *
     * @return \DateTime 
     */
    public function getDateActe()
    {
        return $this->dateActe;
    }

    /**
     * Set descActe
     *
     * @param string $descActe
     * @return ActesJuridiques
     */
    public function setDescActe($descActe)
    {
        $this->descActe = $descActe;

        return $this;
    }

    /**
     * Get descActe
     *
     * @return string 
     */
    public function getDescActe()
    {
        return $this->descActe;
    }

    /**
     * Get idActe
     *
     * @return integer 
     */
    public function getIdActe()
    {
        return $this->idActe;
    }
}

File: Pidev_EGOV/Pi_Egov_Java/src/esprit/egov/entities/ActesJuridiques.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package esprit.egov.entities;

import java.util.Date;
import java.util.Objects;

/**
 *
 * @author macbookpro
 */
public class ActesJuridiques {
   
    private int idActe;
    private String titreActe;
    private Date dateActe;
    private String descActe;

    public ActesJuridiques(String titreActe, Date dateActe, String descActe) {
        this.titreActe = titreActe;
        this.dateActe = dateActe;
        this.descActe = descActe;
    }

    public ActesJuridiques() {
    }

    public int getIdActe() {
        return idActe;
    }

    public String getTitreActe() {
        return titreActe;
    }

    public Date getDateActe() {
        return dateActe;
    }

    public String getDescActe() {
        return descActe;
    }

    public void setIdActe(int idActe) {
        this.idActe = idActe;
    }

    public ActesJuridiques(int idActe) {
        this.idActe = idActe;
    }

    public void setTitreActe(String titreActe) {
        this.titreActe = titreActe;
    }

    public void setDateActe(Date dateActe) {
        this.dateActe = dateActe;
    }

    public void setDescActe(String descActe) {
        this.descActe = descActe;
    }

    @Override
    public String toString() {
        return "ActesJuridiques{" + "idActe=" + idActe + ", titreActe=" + titreActe + ", dateActe=" + dateActe + ", descActe=" + descActe + '}';
    }

    @Override
    public int hashCode() {
        int hash = 7;
        hash = 59 * hash + this.idActe;
        hash = 59 * hash + Objects.hashCode(this.titreActe);
        hash = 59 * hash + Objects.hashCode(this.dateActe);
        hash = 59 * hash + Objects.hashCode(this.descActe);
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final ActesJuridiques other = (ActesJuridiques) obj;
        if (this.idActe != other.idActe) {
            return false;
        }
        if (!Objects.equals(this.titreActe, other.titreActe)) {
            return false;
        }
        if (!Objects.equals(this.dateActe, other.dateActe)) {
            return false;
        }
        if (!Objects.equals(this.descActe, other.descActe)) {
            return false;
        }
        return true;
    }

  
}
